src/LaravelDoctrineSanctumProvider.php

```diff
@@ -130,14 +130,14 @@ private function createAccessTokenRepository(): AccessTokenRepository
         /** @var IlluminateRegistry $registry */
         $registry = $this->app->get('registry');
         $tokenModel = (string) config('sanctum_orm.doctrine.models.token');
-        $deleteUnusedTokensAfter = (int) config('sanctum_orm.delete_unused_tokens_after', 0);
+        $unusedTokenTTL = (int) config('sanctum_orm.unused_token_ttl', 0);
```

Comment:
Volgens mij ben je vergeten om deze daadwerkelijk in de config file aan te passen?